packages/vertexai/package.json

```diff
@@ -37,6 +37,7 @@
     "testsetup": "yarn update-responses && yarn ts-node ./test-utils/convert-mocks.ts",
     "test": "run-p --npm-path npm lint test:browser",
     "test:ci": "yarn testsetup && node ../../scripts/run_tests_in_ci.js -s test",
+    "test:debug": "karma start --single-run",
```

Comment:
I feel that `:debug` is misleading since we are not enabling debugging capabilities, and instead just skip the re-clone

Reply:
I see. How about `:skip-clone`?

Reply:
Yeah I think that would be clearer